### PR TITLE
Resolve mix of British and American spelling

### PR DESCRIPTION
**Pull request by sourishkrout**
Just a simple suggestion to change docs.

### CODE REVIEW

File: docs/current_docs/manuals/developer/typescript/snippets/custom-types/index.ts [hunk 1]
```diff
@@ -34,15 +34,15 @@ class Organization {
...
   @func()
   daggerOrganization(): Organization {
-    const organisation = new Organization()
+    const organization = new Organization()
 
-    organisation.url = "https://github.com/dagger"
-    organisation.repository = [dag.git(`${organisation.url}/dagger`)]
-    organisation.members = [
+    organization.url = "https://github.com/dagger"
+    organization.repository = [dag.git(`${organisation.url}/dagger`)]
```
Comment (vikram-dagger) [inline docs/current_docs/manuals/developer/typescript/snippets/custom-types/index.ts:40]:
```suggestion
    organization.repository = [dag.git(`${organization.url}/dagger`)]
```

**Comment by vikram-dagger** [comment]
Ping @TomChv for approval

**Comment by jedevc** [comment]
Can we also make the fixes in the go/python snippets as well?

**Comment by vikram-dagger** [comment]
> Can we also make the fixes in the go/python snippets as well?

@sourishkrout would you be able to make the above changes?

**Comment by sourishkrout** [comment]
> > Can we also make the fixes in the go/python snippets as well?
> 
> @sourishkrout would you be able to make the above changes?

Sure, fixed all instances that `git grep -in organisation` identified. 👍

**Review by vikram-dagger — APPROVED** [review]

**Comment by vikram-dagger** [comment]
@sourishkrout please fix the DCO error.

Ping @TomChv for review

**Review by TomChv — APPROVED** [review]
LGTM

**Comment by TomChv** [comment]
Fixe DCO and we're good to merge!

**Comment by sourishkrout** [comment]
> Fixe DCO and we're good to merge!

Fixed DCO @vikram-dagger @TomChv 👌

**Pull request merged**